packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -193,64 +155,23 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, any> = {
         !text && {
           color: primaryColor,
           backgroundColor: primaryBackgroundColor,
-          borderWidth: `${pxToRem(primaryBorderWidth)}`,
-          borderStyle: 'solid',
-          borderColor: `${primaryBorderColor}`,
+          borderColor: primaryBorderColor,
+
           ':hover': {
             color: primaryColorHover,
             backgroundColor: primaryBackgroundColorHover,
-            borderColor: primaryBorderColorHover,
           },
-          ...(isFromKeyboard &&
-            !circular && {
-              ':focus': {
-                color: primaryColorFocus,
-                backgroundColor: primaryBackgroundColorFocus,
-                borderColor: primaryBorderColorFocus,
-                ':after': {
-                  content: '""',
-                  position: 'absolute',
-                  top: `-${pxToRem(primaryBorderWidth * 2)}`,
-                  right: `-${pxToRem(primaryBorderWidth * 2)}`,
-                  bottom: `-${pxToRem(primaryBorderWidth * 2)}`,
-                  left: `-${pxToRem(primaryBorderWidth * 2)}`,
-                  borderWidth: pxToRem(primaryBorderWidth),
-                  borderStyle: 'solid',
-                  borderColor: primaryBorderColorFocusIndicator,
-                  borderRadius: borderRadiusFocused,
-                },
-              },
-            }),
-          ...(isFromKeyboard &&
-            circular && {
-              ':focus': {
-                color: primaryColorFocus,
-                backgroundColor: primaryBackgroundColorFocus,
-                borderColor: primaryBackgroundColorFocus,
-                ':after': {
-                  content: '""',
-                  position: 'absolute',
-                  top: '0',
-                  right: '0',
-                  bottom: '0',
-                  left: '0',
-                  borderWidth: pxToRem(primaryBorderWidth),
-                  borderStyle: 'solid',
-                  borderColor: primaryCircularBorderColorFocusIndicator,
-                  borderRadius: circularRadius,
-                },
-              },
-            }),
-          ...(!isFromKeyboard && {
-            ':focus': {
-              ':active': {
-                color: primaryColorActive,
-                backgroundColor: primaryBackgroundColorActive,
-                borderColor: primaryBorderColorActive,
-                boxShadow: 'none',
-              },
-            },
-          }),
+
+          ':focus': {
+            boxShadow: 'none',
+            ...(isFromKeyboard
+              ? {
+                  color: primaryColorFocus,
+                  backgroundColor: primaryBackgroundColorFocus,
+                  ...borderFocusStyles,
+                }
+              : { ':active': { backgroundColor: primaryBackgroundColorActive } }),
```

Comment:
What about the text color: `primaryColorActive`?

Reply:
same response as in: https://github.com/stardust-ui/react/pull/1269#discussion_r282614613

it was not used, it was the same as regular color so no need for it